stdlib/stdlib.mli

```diff
@@ -1271,7 +1271,7 @@ module Oo           = Oo
 module Option       = Option
 module Parsing      = Parsing
 module Pervasives   = Pervasives
-[@@deprecated "Use Stdlib instead.\n\
+[@@deprecated "Use the Stdlib module.\n\
```

Comment:
Here you kept the uppercase and period.  Is that on purpose?

Reply:
Fixed.